test/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeGenerator.java

```diff
@@ -116,6 +116,7 @@
 				makeGeneric(cw, className, "makeValueGeneric", "makeValue", makeValueSig, makeValueGenericSig, fields, makeMaxLocal, isRef);
 			}
 		}
+		addGenerate2DMultiANewArray(cw, className);
```

Comment:
nitpick: please rename to `test2DMultiANewArray(cw, className);` for consistency